Xamarin.Forms.Platform.GTK/Cells/ViewCell.cs

```diff
@@ -51,15 +51,15 @@ private double GetHeightMeasure(Xamarin.Forms.ViewCell viewCell, Gdk.Rectangle a
 			return request.Request.Height;
 		}
 
-		public override void Dispose()
+		public override void Destroy()
```

Comment:
this is the same that we have in our design guideline, i mean the Dispose is never called and the method that is actually called is the Destroy, is this a known bug in gtk-sharp?

Reply:
It's explained in the coments of the commit:
https://github.com/fluendo/Xamarin.Forms/pull/3/commits/d159eccb7c59873d05254d01cd12b9af44148750